controllers/email-controller.js

```diff
@@ -0,0 +1,111 @@
+
+import { SESClient, SendRawEmailCommand } from "@aws-sdk/client-ses";
+import { validationResult } from 'express-validator';
+import { config } from 'dotenv';
+import path from 'path';
+
+config({ path: path.join(process.cwd(), '../.env') });
+
+// Configure AWS
+const ses = new SESClient({
+  region: process.env.AWS_REGION,
+  credentials: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  },
+});
+
+async function sendForm(req, res, next) {
```

Comment:
Please give a more specific name, i.e. sendContactUsForm etc, since we may need to send other emails/forms later